tests/vmi_multus_test.go

```diff
@@ -1185,3 +1204,12 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func getSRIOVInterfaceName(vmi *v1.VirtualMachineInstance) (error, string) {
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
```

Comment:
you should iterate on the status not the spec
and there there isnt iface.SRIOV, but you will find the MAC and the interface name there.